css/style.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700&subset=latin-ext');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+html{
+  font-size: 10px; /* 1rem = 10px */
+}
+body{
+  height: 100vh;
+  font-size: 1.6rem;
+  font-family: 'Lato', sans-serif;
+  background: linear-gradient(90deg, #83EB94 0%, #47D95E 100%);
+  /* background-attachment: fixed; */
+  /* background-repeat: no-repeat; */
+  background-size: contain;
+  color: #fff;
+}
+
+a{
+  text-decoration: none;
+}
+button{
+  font-family: roboto, sans-serif;
+  padding: 0;
+  border: 1px solid;
+  background: none;
+  outline: none;
+  cursor: pointer;
+}
+
+.container{
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  position: relative;
+  margin: 0 auto;
+  padding: 6.3vh 5.3% 0 5.3%;
+  max-width: 100%;
+  height: 100%;
+  width: 1100px;
+  text-align: center;
+}
+
+.header__img{
+  width: 100%;
+  max-width: 688px;
+}
+
+.header__title{
+  margin-top: -2rem;
+  font-size: 6rem;
+  line-height: 10rem;
+  font-weight: 300;
+  letter-spacing: .5rem;
+  text-transform: uppercase;
+}
+
+main{
+  flex-grow: 10;
+}
+
+.contact__title{
+  margin: 4rem auto 1rem;
+  display: block;
+  font-size: 2.4rem;
+}
+
+.contact__input-container{
+  display: inline-block;
+  position: relative;
+}
+
+.contact__input{
+  width: 250px;
+  height: 48px;
+  margin: 2.9rem 1.4rem .9rem;
+  font-family: Lato;
+  font-weight: normal;
+  font-size: 1.6rem;
+  text-indent: 1rem;
+  color: black;
+  border-radius: 9px;
+  box-shadow: inset 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border: none;
+  outline: none;
+}
+
+.contact__input::placeholder{
+  font-weight: 300;
+}
+
+.contact__input:focus{
+  border: 2px solid #5AAF65;
+}
+
+.contact__input:invalid{
+  border: 2px solid red;
+}
+
+.contact__input:invalid ~ .contact__input-warning--error{
```

Comment:
Z tą częścią musisz pokombinować. Nie zadziałało tak jak miało. Od wejścia na stronę wyświetla napis invalid e-mail. Ale idziesz w dobrym kierunku. To może Ci się przydać: https://css-tricks.com/float-labels-css/